src/main/java/Main.java

```diff
@@ -1,6 +1,94 @@
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.text.NumberFormat;
+import java.util.Scanner;
 
 public class Main {
+
+
+    private static String getCurrencyText(BigDecimal price){
+        String roundPrice=price.toBigInteger().toString();
+        String currency;
+        int lastDigit=Integer.parseInt(roundPrice.substring(roundPrice.length()-1));
+        if (lastDigit==1){
+            currency="рубль";
+        }else if (lastDigit==2 || lastDigit==3 || lastDigit==4){
+            currency="рубля";
+        }else {
+            currency="рублей";
+        }
+        return currency;
+    }
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        Check check;
+        Integer countPerson;
+
+        NumberFormat nf = NumberFormat.getCurrencyInstance();
+        DecimalFormatSymbols decimalFormatSymbols = ((DecimalFormat) nf).getDecimalFormatSymbols();
+        decimalFormatSymbols.setCurrencySymbol("");
+        ((DecimalFormat) nf).setDecimalFormatSymbols(decimalFormatSymbols);
+
+
+        //Ввод количества гостей
+        countPerson= EnterData.enterCountOfGuests();
+
+        //Создаем чек
+        check = new Check();
+        check.setCountPerson(countPerson);
+
+        while (true){
+
+            String nameDish;
+            BigDecimal price;
+            Integer person;
+
+            //Ввод названия блюда
+            nameDish= EnterData.enterDish();
+
+            //Ввод цены
+            price= EnterData.enterPrice();
+
+            //Ввод гостя
+            person= EnterData.enterGuest(check.getCountPerson());
+
+            //Приводим к валюте и убираем обозначение валюты
+            Position position=new Position(nameDish,price,person);
+            check.addPosition(position);
+
+
+
+            System.out.println(
+                    String.format("Добавленные товары: %s. %-50s  %20s %s  Гость №%s",
```

Comment:
🍏 Можно упростить вот так:
```suggestion
            System.out.printf("Добавленные товары: %s. %-50s  %20s %s  Гость №%s%n",
```